Documentation/Ctrl/CtrlFormattedLabelUserFunc.rst.txt

```diff
@@ -2,27 +2,51 @@ formattedLabel\_userFunc
 ------------------------
 
 :aspect:`Datatype`
-    string
+   string
 
 :aspect:`Scope`
-    Display
+   Display
 
 :aspect:`Description`
-    Similar to :ref:`label_userFunc <ctrl-reference-label-userfunc>` but allowes to return formatted HTML for the label
-    **and used only for the labels of inline (IRRE) records**. The referenced user function may receive optional arguments using the
-    :ref:`formattedLabel_userFunc_options <ctrl-reference-formattedlabel-userfunc-options>` property.
+   Similar to :ref:`label_userFunc <ctrl-reference-label-userfunc>` but allows
+   to return formatted HTML for the label **and used only for the labels of
+   inline (IRRE) records**. The referenced user function may receive optional
+   arguments using the :ref:`formattedLabel_userFunc_options
+   <ctrl-reference-formattedlabel-userfunc-options>` property.
 
-    **Example from table "sys_file_reference"**
+   .. tip::
 
-    .. code-block:: php
+      Read more about :ref:`Inline Relational Record Editing (IRRE)
+      <columns-inline>`.
 
-        'formattedLabel_userFunc' => TYPO3\CMS\Core\Resource\Service\UserFileInlineLabelService::class . '->getInlineLabel',
-        'formattedLabel_userFunc_options' => [
-            'sys_file' => [
-                'title',
-                'name'
-            ]
-        ],
+   **Example from table "sys_file_reference"**
 
-    See class :php:`TYPO3\CMS\Core\Resource\Service\UserFileInlineLabelService`
-    for how such a user function should be designed and how the options are used.
+   .. code-block:: php
+
+      'formattedLabel_userFunc' => TYPO3\CMS\Core\Resource\Service\UserFileInlineLabelService::class . '->getInlineLabel',
+      'formattedLabel_userFunc_options' => [
+         'sys_file' => [
+            'title',
+            'name'
+         ]
+      ],
+
+   The passed argument is an array which contains the following information
+   about the record for which to get the title::
+
+      $params['table'] = $table;
+      $params['row'] = $row;
+      $params['isOnSymmetricSide'] = $isOnSymmetricSide;
+      $params['options'] = $options;
+      $params['parent'] = [
+         'uid' => $uid,
+         'config' => $config,
+      ];
```

Comment:
Instead of a copy from the original source which explains almost nothing we should instead mention the available keys and their purpose.